diff -urN acpi_call-1.1.0.orig/acpi_call.c acpi_call-1.1.0/acpi_call.c
--- acpi_call-1.1.0.orig/acpi_call.c	2013-07-05 18:06:13.000000000 +0200
+++ acpi_call-1.1.0/acpi_call.c	2019-12-29 00:25:36.948131987 +0100
@@ -5,8 +5,11 @@
 #include <linux/version.h>
 #include <linux/proc_fs.h>
 #include <linux/slab.h>
+#if LINUX_VERSION_CODE < KERNEL_VERSION(4, 14, 0)
 #include <asm/uaccess.h>
-#include <acpi/acpi.h>
+#endif
+#include <linux/acpi.h>
+#include <linux/uaccess.h>
 
 MODULE_LICENSE("GPL");
 
